shoop/core/models/_shipments.py

```diff
@@ -45,6 +45,7 @@ class Shipment(models.Model):
     description = models.CharField(max_length=255, blank=True, verbose_name=_("description"))
     volume = MeasurementField(unit="m3", verbose_name=_("volume"))
     weight = MeasurementField(unit="kg", verbose_name=_("weight"))
+    shipment_key = models.CharField(max_length=64, unique=True, verbose_name=_("shipment key"))
```

Comment:
I think this should be called `identifier`, to keep in line with all of the other `identifier`s in Shoop. (Also, you could use `InternalIdentifierField`).